include/deal.II/dofs/dof_tools.h

```diff
@@ -2348,13 +2351,16 @@ namespace DoFTools
    * @param support_points A map that for every locally relevant DoF index
    * contains the corresponding location in real space coordinates. Previous
    * content of this object is deleted in this function.
+   * @param component_mask An optional component mask that restricts the
+   * components from which the support points are extracted
```

Comment:
Copy this sentence to the other places. Add a period to the end of the sentence.